src/consensus/aft/test/driver.cpp

```diff
@@ -65,15 +93,11 @@ int main(int argc, char** argv)
         assert(items.size() == 1);
         driver->dispatch_all_once();
         break;
-      case shash("state_all"):
-        assert(items.size() == 1);
-        driver->state_all();
-        break;
       case shash("replicate"):
-        assert(items.size() == 4);
+        assert(items.size() == 3);
         data = std::make_shared<std::vector<uint8_t>>(
-          items[3].begin(), items[3].end());
-        driver->replicate(items[1], stoi(items[2]), data);
+          items[2].begin(), items[2].end());
+        driver->replicate(items[1], data);
```

Comment:
This `replicate` step previously tried to dictate the node and index to replicate on and at. I don't think this is a helpful abstraction - it is only valid to `replicate` on a node if they currently believe they are a primary, and the index should always be monotonic. I've changed this to take the `term` to replicate in (its valid and interesting for a partitioned primary, who is still in term X, to replicate _in_ term X, while the other nodes have advance to term Y). Additionally this term can be the special value `latest` to replicate something in the last term-with-a-primary, which is what the scenario generator wants to do.